OmniKit/MessageTransport/MessageBlocks/PodInfoFaultEvent.swift

```diff
@@ -96,27 +97,28 @@ public struct PodInfoFaultEvent : PodInfo, Equatable {
 
 extension PodInfoFaultEvent: CustomDebugStringConvertible {
     public typealias RawValue = Data
+    
     public var debugDescription: String {
         return [
             "## PodInfoFaultEvent",
             "* rawHex: \(data.hexadecimalString)",
             "* podProgressStatus: \(podProgressStatus)",
             "* deliveryStatus: \(deliveryStatus.description)",
-            "* insulinNotDelivered: \(insulinNotDelivered) U",
-            "* podMessageCounter: \(podMessageCounter)",
-            "* unknownPageCode: \(unknownPageCode.hexadecimalString)",
+            "* insulinNotDelivered: (%.2f, \(insulinNotDelivered)) U",
```

Comment:
I don't think this is doing what you are intending. It will render like `* totalInsulinDelivered: (%.2f, 0.0) U`

Reply:
made an extension for this.